src/TicketMap/index.test.tsx

```diff
@@ -0,0 +1,355 @@
+import React from 'react'
+import { shallow, ShallowWrapper } from 'enzyme'
+
+import TicketMap from './index'
+import { State, Props, Manifest } from './types'
+import Tooltip from '../Tooltip'
+import Actions from '../Actions'
+import { TicketGroup } from '../types'
+
+describe('TicketMap', () => {
+  let wrapper: ShallowWrapper<Props, State, TicketMap>
+  let props: Props
+  let manifestResponse: [string, { status: number }]
+  let mapResponse: [string, { status: number }]
+  let mapRoot: HTMLDivElement
+  let manifest: Manifest
+  let ticketGroup: TicketGroup
+  let fillSectionSpy: jest.SpyInstance
+
+  const mountComponent = async (responses = [mapResponse, manifestResponse]) => {
+    fetchMock.mockResponses(...responses)
+    wrapper = shallow(<TicketMap {...props} />)
+    wrapper.instance().mapRoot = { current: mapRoot }
+    fillSectionSpy = jest.spyOn(wrapper.instance(), 'fillSection')
+    await new Promise(setImmediate)
+  }
+
+  beforeEach(() => {
+    ticketGroup = {
+      'tevo_section_name': 'Foo Bar',
+      'retail_price': 10
+    }
+    mapRoot = document.createElement('div')
+    manifest = {
+      sections: {
+        'foo bar': {
+          'zone_name': ''
+        }
+      }
+    }
+    mapResponse = [ '<svg><path data-section-id="foo bar" /></svg>', { status: 200 } ]
+    manifestResponse = [ JSON.stringify(manifest), { status: 200 } ]
+    props = {
+      venueId: '1',
+      configurationId: '2'
+    }
+  })
+
+  afterEach(() => {
+    fetchMock.mockClear()
+  })
+
+  it('fetches the map associated with the venue and config IDs', async () => {
+    await mountComponent()
+    expect(fetchMock).toHaveBeenCalledWith('https://maps.ticketevolution.com/1/2/map.svg')
+  })
+
+  it('renders a not found image if the map fetch fails', async () => {
+    jest.spyOn(global.console, 'error').mockImplementationOnce(() => {})
+    await mountComponent([[ mapResponse[0], { status: 404 } ]])
+    expect(wrapper.state('mapNotFound')).toBe(true)
+    expect(wrapper.find('img').exists()).toBeTruthy()
+    expect(wrapper.text()).toContain('Seating chart not available.')
+  })
+
+  it('fetches the manifest associated with the venue and config IDs', async () => {
+    await mountComponent()
+    expect(fetchMock).toHaveBeenCalledTimes(2)
+    expect(fetchMock).toHaveBeenCalledWith('https://maps.ticketevolution.com/1/2/manifest.json')
+  })
+
+  it('renders a tooltip', async () => {
+    await mountComponent()
+    expect(wrapper.find(Tooltip).exists()).toBeTruthy()
+  })
+
+  it('renders an Action menu', async () => {
+    await mountComponent()
+    expect(wrapper.find(Actions).exists()).toBeTruthy()
```

Comment:
Now that we have `jest-enzyme` installed, we can shorten this to 
```suggestion
    expect(wrapper.find(Actions)).toExist()
```